f3/granite.go

```diff
@@ -798,14 +795,14 @@ func (i *instance) alarmAfterSynchrony() float64 {
 	}
 
 	if i.round >= 5 {
-		delta *= i.deltaRate
-		i.deltaRate *= i.config.DeltaRate
+		delta = delta * math.Pow(i.config.DeltaBackOffExponent, float64(i.round-4))
 	}
 
 	timeout := i.host.Time() + delta
+
 	if i.round > 0 && i.round%5 == 0 && i.phase == CONVERGE_PHASE {
```

Comment:
Carrying conversation from https://github.com/filecoin-project/go-f3/pull/82#discussion_r1491628191

> Would it make more sense in your opinion that we signal with a parameter in the call from CONVERGE that the round is starting?

Yes, I think a parameter like `waitForNextClockBoundary` or similar would make this clearer here, and at the call sites.

> I do realize now that the spec does. not specify every 5th round but specifically in the 5th round. I think it makes more sense every 5th round, so should update the spec instead in this case.

This needs more thought at the spec level. I don't think every 5th round makes sense, though I also don't think once is particularly good either. I'll start a discussion in Slack, let's resolve something there, then come here to fix it.

Reply:
waitForNextClockBoundary done, and for the slack discussion I answered there.